src/Math-ODE/PMAB2Stepper.class.st

```diff
@@ -15,9 +15,10 @@ PMAB2Stepper class >> order [
 
 { #category : #stepping }
 PMAB2Stepper >> doStep: aState prevState: prevState time: t [
-	self stepSize isNil
-		ifTrue: [ self error: 'step size required by stepper' ].
-	^ (self stepSize / 2) * (3 * (system state: aState time: t) - (system state: prevState time: t - self stepSize)) + aState
+
```

Comment:
The doStep, could be renamed to

`stepFrom: previousState to: newState inStepsOf: stepSize` 

This seems more intention-revealing and almost fluent. Is this really what's happening? Are we stepping from one state to another in steps? Seems so.